lib/grammar-utils.js

```diff
@@ -56,7 +56,7 @@ export default {
   // Public: Format args for cmd or bash, depending on the current OS
   formatArgs(command) {
     if (os.platform() === 'win32') {
-      return [`/c ${command.replace(/['"]/g, '')}`];
+      return [`/c ${command.replace(/[']/g, '')}`];
```

Comment:
Was this the cause of the [issue](#1455)… removing double quotes? If so, then the `[]` brackets can be omitted, so just: `return [`/c ${command.replace(/\'/g, '')}`];`